web/client-backplane/src/main/java/io/deephaven/javascript/proto/dhinternal/io/deephaven/proto/table_pb/SeekRowResponse.java

```diff
@@ -1,6 +1,3 @@
-/**
- * Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
- */
```

Comment:
if you run the licenseFormat task it will not remove these, and will add them for new files